docs/src/misc.rst

```diff
@@ -15,15 +15,12 @@ Data types
 
     Buffer data type.
 
-.. c:type:: uv_malloc_func
+.. c:type:: uv_malloc_config
```

Comment:
Please call this uv_allocator_t and document the members using the right syntax.